esgfpub/esgfpub/version.py

```diff
@@ -1 +0,0 @@
-__version__ = '1.0.0'
```

Comment:
This file is not necessary because `__version__` is usually defined in `__init__.py`.